### PR TITLE
Calculates prior prob and ENIGMA class for variants in splice donor/acceptor regions

### PR DESCRIPTION
**Pull request by ccdlott**

### CODE REVIEW

**Comment by zfisch** [comment]
Ah, yes it is a problem. Are you comfortable making your way back to where you need to be and updating the PR? If not, let me know and I can help you with it tomorrow afternoon.

**Comment by ccdlott** [comment]
Done, I made the changes you requested and reverted back to a previous commit so I now only see my 8 commits for this branch

File: pipeline/splicing/calcVarPriors.py [hunk 1]
```diff
@@ -369,11 +381,29 @@ def varInSpliceDonor(variant):
             return True
     return False
 
+def getVarSpliceDonorBounds(variant):
+    '''
+    Given a variant, checks if variant is in a splice donor region
+    If variant is in a splice donor, returns a dictionary with donor region boundaries where variant is located
+    '''
+    inSpliceDonor = varInSpliceDonor(variant)
+    if inSpliceDonor == True:
```
Comment (zfisch) [inline pipeline/splicing/calcVarPriors.py]: a simpler way to write this is:

`if varInSpliceDonor(variant):`

(no need for inSpliceDonor variable)

File: pipeline/splicing/calcVarPriors.py [hunk 2]
```diff
@@ -369,11 +381,29 @@ def varInSpliceDonor(variant):
...
+        varGenPos = int(variant["Pos"])
```
Comment (zfisch) [inline pipeline/splicing/calcVarPriors.py]: this whole block from line 391-398 is also in `varInSpliceDonor`, `varInSpliceAcceptor`, and `getVarSpliceAcceptorBounds`  -- can you put it into a function to keep DRY? You'll need to take a couple arguments to make it flexible enough to handle donors and acceptors :)

Comment (ccdlott) [reply pipeline/splicing/calcVarPriors.py]: I combined varInSpliceDonor and varInSpliceAcceptor into one function and getVarSpliceDonorBounds and getVarSpliceAcceptorBounds into another function. Is that enough to keep it DRY?

**Pull request merged**